relayer/src/event/rpc.rs

```diff
@@ -143,22 +148,19 @@ pub fn get_all_events(
 
             for abci_event in &tx_result.result.events {
                 if query == queries::ibc_client().to_string() {
```

Comment:
I think we need something like -> 
```rust
if let Ok(ibc_event) = IbcEvent::try_from(abci_event) {
  fn is_type_client(ev: &IbcEvent) -> bool {
      matches!(
          ev,
          IbcEvent::CreateClient(_)
              | IbcEvent::UpdateClient(_)
              | IbcEvent::UpgradeClient(_)
              | IbcEvent::ClientMisbehaviour(_)
      )
  }

  if query == queries::ibc_client().to_string() && is_type_client(&ibc_event) {
      tracing::trace!("extracted ibc_client event {}", client_event);
      events_with_height.push(IbcEventWithHeight::new(ibc_event, height));
  } /* else if ... */
}
```

Reply:
[12484d9](https://github.com/informalsystems/ibc-rs/pull/2542/commits/12484d9dd8eca10d554f8c52e5132e2285961f99)